assets/javascripts/kitten/components/links/link-list/link-list.stories.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { select, boolean } from '@storybook/addon-knobs'
+import { LinkList } from './index'
+import { Grid, GridCol } from '../../grid/grid'
+
+const colorOptions = {
+  Light: 'light',
+  Dark: 'dark',
+}
+
+const marginOptions = {
+  Double: 'double',
+  Triple: 'triple',
+}
+
+const weightOptions = {
+  Regular: 'regular',
+  Light: 'light',
+}
+
+export default {
+  title: 'Links/Link-list',
+  component: LinkList,
+}
+
+export const SimpleList = () => {
+  return (
+    <LinkList
+      color={select('Color', colorOptions, 'dark')}
+      itemMargin={select('Item margin', marginOptions, 'double')}
+      weight={select('Weight', weightOptions, 'regular')}
+      margin={boolean('Margin', true)}
+      items={[
+        { key: '1', item: 'Item 1', href: "#", active: true },
+        { key: '2', item: 'Item 2', href: "#" },
+        { key: '3', item: 'Item 3', href: "#" },
+      ]}
+    />
+  )
+}
+
+export const WithBackgroundDark = () => {
+  return (
+    <Grid>
```

Comment:
Je pense qu'on peut changer la couleur de fond de la Grid en changeant la prop `color`.

On pourrait faire ça avec :

```suggestion
    <Grid className={select('Color', colorOptions, 'light') && 'has-overrides dark-background'}>
```

Et parallèlement, on fait un fichier `.storybook/stylesheets/custom/_background.scss` qui contient le code :

```scss
.has-overrides {
  &.dark-background {
    background-color:  k-map-fetch($k-colors, 'primary-4');
  }
}
```

(sans oublier d'ajouter le lien dans `.storybook/stylesheets/app-kitten.scss` ligne 18 sous la forme `@import 'custom/background';`)